### ISSUE TITLE
feat: Android version

### ISSUE THREAD

**Issue by konikvranik** [music-assistant/companion#56]
I'd like to vote for the Android version of this app too.

**Comment by arctixdev** [comment]
Yes I agree that this would be very useful. I don't have time to work on it at the moment but anyone is welcome to pick up the work 🙂 Setting up android with tauri is actually super easy, the hardest part is to get audio streaming to work

**Comment by cociweb** [comment]
+1

additionally with AAuto support.
